portfolio/src/main/webapp/script.js

```diff
@@ -31,3 +31,10 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+async function addMotivationalMessage(){
+  const responseFromServer = await fetch('/hello');
```

Comment:
nit: Add a space before '{ . So () { instead of (){. This is true every where you use curly brackets.

Reply:
Done